src/App.js

```diff
@@ -72,84 +125,107 @@ const App = () => {
     setState({ showMsg: false });
   };
 
+  const clearProject = () => {
+    setState({ selectedProject: '' });
+    sessionStorage.removeItem('project');
+  };
+
+  const setSelectedProject = (selectedProject) => {
+    setState({ selectedProject });
+  };
+
   return (
     <MuiThemeProvider theme={theme}>
-      <AppBar position="static">
+      <AppBar position='static'>
         <Toolbar>
-          <Typography variant="h6" color="inherit">
+          <img
+            src='sprintcompass_logo.png'
+            className={classes.headerLogo}
+            alt='fast compass logo'
+          />
+          <Typography variant='h6' color='inherit'>
             Sprint Compass
           </Typography>
+          {(useAuth && !state.loggedIn) || (
+            <FormControl className={classes.projectChoice} color='secondary'>
+              <Select
+                value={state.selectedProject}
+                color='secondary'
+                displayEmpty
+                onChange={handleSelectProject}
+                label='Selected Project'>
+                <MenuItem value=''>
+                  <em>Select A Project</em>
+                </MenuItem>
+                {state.projects.map(({ projectName }) => (
+                  <MenuItem value={projectName}>{projectName}</MenuItem>
+                ))}
+                <MenuItem
+                  value='addNewProject'
+                  component={Link}
+                  to='/projectdetails'
+                  onClick={clearProject}>
+                  + Add A New Project
+                </MenuItem>
+              </Select>
+            </FormControl>
+          )}
           {(useAuth && !state.loggedIn) || ( // Only show dropdown menu once logged in
```

Comment:
I thought you joined components with && just because the content in the '( )' will always resolve to true because it isnt undefined

Reply:
if useAuth === false and the user is not logged in then the first bit resolves to true and the component isn't rendered, but if we're using auth and they ARE logged in (state.loggedIn === false), the () returns false and it does go to the component after ||

Reply:
I had to do it this way because if it was an &&, the latter would never be rendered if useAuth was false (when we aren't locking it behind log in)